packages/api/src/clickhouse/__tests__/clickhouse.test.ts

```diff
@@ -1,4 +1,5 @@
 import * as clickhouse from '..';
+import { describe, beforeEach, jest, it, expect } from '@jest/globals';
```

Comment:
I don't think we need to import these from globals. did you hit any issues on local ?